examples/catalog.json

```diff
@@ -13,19 +13,43 @@
       "rel": "child",
       "href": "./usda-cdl/collection.json",
       "type": "application/json",
-      "title": "USDA Cropland Data Layer (CDL)"
+      "title": "USDA Cropland Data Layers (CDLs)"
     },
     {
       "rel": "child",
-      "href": "./usda-cdl-cultivated/collection.json",
+      "href": "./usda-cdl/collection.json",
+      "type": "application/json",
+      "title": "USDA Cropland Data Layers (CDLs)"
+    },
+    {
+      "rel": "child",
+      "href": "./usda-cdl/collection.json",
+      "type": "application/json",
+      "title": "USDA Cropland Data Layers (CDLs)"
+    },
```

Comment:
Looks like we have multiple links to the same collection ... in fact, we shouldn't need a top-level catalog at all, since we only have one collection.

Reply:
Catalog level removed.  The list of items within collection.json doesn't seem correct though.

Reply:
Looks good to my eye but maybe I'm missing something.